www/docs/src/content/docs/0.1.0-beta.5/config-reference/dashboard.md

```diff
@@ -0,0 +1,166 @@
+---
+title: dashboardConfig
+description: A reference page for dashboard
+sidebar:
+  order: 4
+slug: 0.1.0-beta.5/config-reference/dashboard
+---
+
+`dashboardConfig` option let's customize the dashboard for the `astroStudioCMS`. This is used to setup your dashboard data.
```

Comment:
```suggestion
`dashboardConfig` option allows you to customize the dashboard for the `astroStudioCMS`. This is used to setup your dashboard data.
```

Reply:
This is the old version of the docs.... (beta.5)

Reply:
Updated the Latest version of the docs on [d869f86](https://github.com/astrolicious/studiocms/pull/281/commits/d869f869670c49fbd9c8f357a32ed15fe00a20c5)